scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,19 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+os_type=darwin
+case "$OSTYPE" in
+  darwin*)
+    os_type=darwin
+    ;;
+  linux*)
+    os_type=ubuntu
```

Comment:
```
$ docker run -i -t heron-ubuntu14.04 /bin/bash
root@45c2337fdea9:/# ls -l /etc/*-release
-rw-r--r-- 1 root root 105 Feb 16 16:29 /etc/lsb-release
-rw-r--r-- 1 root root 249 Feb 16 16:43 /etc/os-release
root@45c2337fdea9:/# cat /etc/os-release
NAME="Ubuntu"
VERSION="14.04.4 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.4 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"

$ docker run -i -t heron-centos7 /bin/bash
[root@ea2d85dbf2af /]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

On our internal centos hosts though, `/etc/os-release` doesn't exist but `/etc/centos-release` does, so we'd need to also check for that. Distro-detection would be a good thing to put in it's own scripts, since I suspect it might be useful to others as well.
```
$ cat /etc/centos-release
Nest Enterprise Linux release 5.11 (Final) (Based on CentOS release 5.11)
```